### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by taotieren**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,6 +84,14 @@ Current firmware version: 2.11 (aka. v31).
 > On Linux, you should install libudev and libusb development lib first.
 > Like `sudo apt install libudev-dev libusb-1.0-0-dev` on Ubuntu.
 
+### Arch Linux
+
+Arch Linux can install [wlink-git](https://aur.archlinux.org/packages/wlink-git) via the AUR.
+
+```bash
+yay -Syu wlink
```
Comment (andelf) [inline README.md:92]: Is this wlink-git?

Comment (taotieren) [reply README.md:92]: The package name wlink is provided in [wlink-git](https://aur.archlinux.org/packages/wlink-git) and can be installed via `yay -Syu wlink` or `yay -Syu wlink-git`. However, wlink is not released at this time and is reserved for future releases.

File: README.md [hunk 2]
```diff
@@ -84,6 +84,14 @@ Current firmware version: 2.11 (aka. v31).
 > On Linux, you should install libudev and libusb development lib first.
 > Like `sudo apt install libudev-dev libusb-1.0-0-dev` on Ubuntu.
 
+### Arch Linux
+
+Arch Linux can install [wlink-git](https://aur.archlinux.org/packages/wlink-git) via the AUR.
```
Comment (andelf) [inline README.md:89]:
```suggestion
Arch Linux users can install [wlink-git](https://aur.archlinux.org/packages/wlink-git) via the AUR.
```

Comment (taotieren) [reply README.md]: Thanks!

**Pull request merged**